sn_cli/src/subcommands/files.rs

```diff
@@ -260,6 +260,11 @@ async fn upload_files(
             .await
         {
             Ok((cost, new_balance)) => (cost, new_balance),
+            Err(ClientError::Transfers(WalletError::Transfer(
```

Comment:
I think within the pay_for_chunks function, we shall have something as bellow to carry out an early check.
To avoid the overall chunk split & initial pamenet calculate & upload attempts ?
```rust
pub async fn pay_for_chunks(&self, chunks: Vec<XorName>) -> Result<(NanoTokens, NanoTokens)> {
    let mut wallet_client = self.wallet()?;
    let old_balance = wallet_client.balance();
    if old_balance.as_nano() < chunks.len() {
        return Err(TransfersError::NotEnoughBalance(old_balance, NanoTokens::from(chunks.len() * 100));
    }
    ......
}
```